docker-compose.dev.yml

```diff
@@ -19,3 +28,11 @@ services:
     ports:
       - "3306:3306"
     restart: unless-stopped
+  
+  databasebackup:
+    container_name: micro-pkc-databasebackup-dev
+    environment:
+      USERNAME: wikiuser
+      PASSWORD: "example"
+      CRON_SCHEDULE: "0 */2 * * *"
```

Comment:
It would be helpful to have a human-readable/understandable format of this cron schedule.